linera-base/src/version_info.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::borrow::Cow;
+
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    async_graphql::SimpleObject,
+    serde::Deserialize,
+    serde::Serialize,
+)]
+#[non_exhaustive]
+/// The version info of a build of Linera.
+pub struct VersionInfo {
+    /// The crate version
+    pub ver: Cow<'static, str>,
```

Comment:
I was originally using `&'static str`.  With a `String` we can't make it a `static` or `const` because `String` requires allocation — I introduced `Cow` just because we have to be able to deserialize it.  I'd usually default to `Cow<str>` here to avoid unnecessary copies everywhere — it simplifies usage a little.

Fully agree on the naming though.